src/main/java/dev/ebrydeu/spring_boot_library/domain/entities/Message.java

```diff
@@ -0,0 +1,30 @@
+package dev.ebrydeu.spring_boot_library.domain.entities;
+
+import jakarta.persistence.*;
+import lombok.*;
+import java.util.UUID;
+
+@Data
+@Entity
+@Table (name = "MESSAGE")
```

Comment:
Use [lowercase](https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html) instead.
```suggestion
@Table (name = "message")
```